ckanext/string_to_location/location_mapper_log_writer.py

```diff
@@ -38,7 +38,8 @@ def _log(self, log_level, message, state, context):
                 'key': 'location_mapper'
             })
             task = existing_task
-            task['state'] = state
+            if not existing_task['state'] == 'complete':
+                task['state'] = state
```

Comment:
Should this be "If the task state is not passed, use the existing_task state? If the existing_task state is not set and no state is passed, use `processing`"?